demo-notebooks/guided-demos/2_basic_jobs.ipynb

```diff
@@ -116,7 +116,24 @@
    "id": "83d77b74",
    "metadata": {},
    "source": [
-    "First, let's begin by submitting to Ray, training a basic NN on the MNIST dataset:"
+    "First, let's begin by submitting to Ray, training a basic NN on the MNIST dataset:\n",
+    "\n",
+    "NOTE: To test this demo in an air-gapped/ disconnected environment alter the training script to use a local dataset.\n",
+    "First we must download the MNIST dataset. We've included a helper script to do this for you. \n",
+    "\n",
+    "Uncomment the following cell to download the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## download the dataset\n",
+    "# ! python -m download_mnist_datasets.py\n",
```

Comment:
This cell is a bit tricky, as it can't be ran on disconnected cluster.
User needs to run the script locally and copy the result folder into cluster.

Reply:
I've updated it. Removed the cell and just gave some instructions on how do to it and the path to place it in instead 👍